Dominant Color Extraction/readme.md

```diff
@@ -0,0 +1,38 @@
+# DOMINANT COLOR DETECTION FROM IMAGE 
+
+## Introduction
+```
+1.) Finding Dominant Colour: Finding Colour with
+Kmeans and
+giving appropriate colors to the pixel/data points
+of the image.
+2.) Analysis different K in K-means : Different Ks can
+give totally different results, and usually, there is
+one K which is consistent with the data points,
+the actual K.
+3.) Good Segmentation(Colour Extraction) : Given
+an image in generic viewpoint, we wish to
+produce the best possible segmentation, by this
+we mean that most people would choose this
+segmentation if asked to make one, and it would
+make a good precursor for object recognition.
+
+```
+## How to use it :
+1. Download or clone the repository
+2. Install Required Libraries
```

Comment:
As you have already added ```requirements.txt``` you can ask user to just run ```pip3 install -r requirements.txt``` to install all the dependencies at once.

Reply:
okay changed

Reply:
I added it to so user can do it both ways